css-three-column-layout/styles.css

```diff
@@ -0,0 +1,56 @@
+div {
+  font-family: 'Lato', sans-serif;
+}
+
+h1,
+h3,
+h4 {
+  text-align: center;
+}
+
+h1 {
+  font-weight: 500;
+  font-size: 29px;
+}
+
+h3 {
+  font-size: 21px;
+  font-weight: 400;
+}
+
+.circle {
+  background-color: white;
+  border-radius: 50%;
+  width: 110px;
+  height: 110px;
+  border: 2px solid #dddede;
+}
+
+img {
+  padding-top: 25px;
+  max-width: 60px;
+}
+
+h4 {
+  max-width: 248px;
+  color: #dadcdd;
+  font-weight: 400;
+}
+
+.column {
+  padding: 10px 30px;
+  max-width: 33%;
+}
+
+#container {
+  padding: 10px;
+  color: white;
+  background-color: #1996da;
+}
+
+#row {
+  display: flex;
+  margin: auto;
+  max-width: 1000px;
+  justify-content: center;
+}
```

Comment:
You shouldn't use `id`s for CSS, you will want to stick to classes, since they are reusable. Also, the `row` class should only have the styling `display: flex`, you should apply the `max-width` and `margin` to the `container` class, that is the `container` class's purpose.